transfocate/lens.py

```diff
@@ -47,6 +47,8 @@ class Lens(InOutPVStatePositioner):
     # Default configuration attributes. Read attributes are set correctly by
     # InOutRecordPositioner
     _default_configuration_attrs = ['_sig_radius', '_sig_z']
+    # Signal for requested focus
+    _req_focus = C(EpicsSignal, ':REQ_FOCUS')
```

Comment:
We should probably refactor this at some later date to not have _leading _underscores _on _every _signal _name